db/db.go

```diff
@@ -227,6 +245,31 @@ func QueryTxs(conn *pgxpool.Conn, events types.StringEvents, height uint64, limi
 		return nil, err
 	}
 	defer rows.Close()
+	return parseRows(rows, limit)
+}
+
+func QueryTxsWithKeyword(conn *pgxpool.Conn, events types.StringEvents, limit uint64, offset uint64, order Order, keyword string) ([]*types.TxResponse, error) {
+	sql := fmt.Sprintf(`
+		SELECT tx FROM txs
+		WHERE events @> $1
+		AND jsonb_to_tsvector('english', tx #> '{"tx", "body", "messages", 0, "record", "contentMetadata"}' , '["string"]') 
```

Comment:
It hasn't been fully tested. For my experience, it can still find the keyword. But I'm not sure if it is the best result or whether it has missed something.

Reply:
removed fulltext search